src/honkalculate.template.html

```diff
@@ -57,7 +57,8 @@
         <span title="Select the calculator's mode of function.">
             <input class="mode visually-hidden" type="radio" name="mode" id="all-vs-one" /><label class="btn btn-wide btn-right right" for="all-vs-one">All vs One</label>
             <input class="mode visually-hidden" type="radio" name="mode" id="one-vs-all" /><label class="btn btn-wide btn-mid right" for="one-vs-all">One vs All</label>
-            <input class="mode visually-hidden" type="radio" name="mode" id="one-vs-one" /><label class="btn btn-wide btn-left right" for="one-vs-one">One vs One</label>
+            <input class="mode visually-hidden" type="radio" name="mode" id="one-vs-one" /><label class="btn btn-wide btn-mid right" for="one-vs-one">One vs One</label>
+            <input class="mode visually-hidden" type="radio" name="mode" id="randoms" /><label class="btn btn-wide btn-left right" for="randoms">Random Battles</label>
```

Comment:
Please sure this toggle doesn't distort at different browser sizes (you'll probably need to tweak the CSS - i think it already distorts on occasion even without the new toggle)